Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -183,9 +183,9 @@ def prepare(self,mReader, directories, mode='init'):
             wmsplit['TTbar_13_ID']=1
             wmsplit['SingleMuPt10FS_ID']=1
             wmsplit['TTbarFS_ID']=1
-            wmsplit['RECODR2_50nsreHLT']=1
-            wmsplit['RECODR2_25nsreHLT']=1
-            wmsplit['RECODR2_2016sreHLT']=1
+            wmsplit['RECODR2_50nsreHLT']=5
+            wmsplit['RECODR2_25nsreHLT']=5
+            wmsplit['RECODR2_2016reHLT']=5
```

Comment:
@davidlange6 good idea! let me think how to implement this idea after i finished the tasks for reHLT. i will either just implement it, or will come back to in case i see a show stopper.